ros2pkg/ros2pkg/api/create.py

```diff
@@ -18,7 +18,9 @@
 
 import em
 
-RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', 'resource')
+from ros2pkg.api import get_prefix_path
+
+RESOURCE_PATH = os.path.join(get_prefix_path('ros2pkg'), 'share', 'ros2pkg', 'resource')
```

Comment:
Is this an absolute path ?
If so, `template_path` line 71 can just get rid of `__file__` as well and just use the resource path directly

Reply:
yeah I think __file__ maybe didn't belong regardless. will fix

Reply:
https://github.com/ros2/ros2cli/pull/87/commits/1f7b17eed2fbd5bcbe21589035647a1884baf561